generators/profilegen/src/profile_templates.rb

```diff
@@ -9,25 +9,38 @@
 # during the generation of profile sdks
 #
 class ProfileTemplates
+  Profile_gen_resources_path = 'generators/profilegen/src/resources'
   #
   # Module template to generate the individual module
   # files per RP (such as ComputModule, NetworkModule, etc)
   #
-  def self.module_template
-    File.read(ProfileTemplateFiles::MODULE_TEMPLATE)
+  def self.module_template(sdk_path, individual)
+    if(individual)
+      File.read("#{sdk_path}/#{Profile_gen_resources_path}/#{ProfileTemplateFiles::Individual::MODULE_TEMPLATE}")
```

Comment:
is there a reason to construct more pieces of the path here, instead of having them defined directly in the constant?